hybrid-analysis/info.json

```diff
@@ -0,0 +1,1137 @@
+{
+    "name": "hybrid-analysis",
+    "version": "1.1.1",
```

Comment:
you want to call this 2.0.0 since the APIs are not compatible with older versions?